02SimpleArraySum.rb

```diff
@@ -0,0 +1,21 @@
+require 'json'
```

Comment:
You can keep just the function (line 4 to 7). Rest of the codes are not required. Its auto generated in hackkerrank.